packages/ffe-datepicker-react/src/calendar/Calendar.js

```diff
@@ -163,16 +162,27 @@ export default class Calendar extends Component {
     }
 
     renderDate(date, index) {
-        if (date.isLead) {
-            return <LeadDate key={date.date} date={date} />;
+        const { calendar } = this.state;
+
+        if (date.isNonClickableDate) {
+            return <NonclickableDate key={date.date} date={date} />;
         }
+
+        const forceFocus =
+            document.activeElement !== this.nextMonthButtonElementRef.current &&
+            document.activeElement !== this.prevMonthButtonElementRef.current;
+
         return (
-            <ActiveDate
+            <ClickableDate
                 date={date}
+                month={calendar.focusedMonth()}
                 headers={`header__${this.datepickerId}__${index}`}
                 key={date.date}
                 onClick={this.mouseClick}
-                forceFocus={this.forceDateFocus}
+                forceFocus={forceFocus}
+                language={this.props.language}
+                activeRef={date.isFocus ? this.activeDateRef : undefined}
+                isBlured={this.state.isFocusingHeader}
```

Comment:
Denne prop'en kan sikkert også kalles for `isFocusingHeader`. `isBlured` er et litt intetsigende navn. Spesielt når vi inni `ClickableDate` har `isFocus && !isBlured`. Da blir det mer forståelig `isFocus && !isFocusingHeader` syns jeg :)